app-text/dblatex/dblatex-0.3.11.ebuild

```diff
@@ -0,0 +1,62 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="7"
+PYTHON_COMPAT=( python3_{6,7} )
+
+inherit distutils-r1
+
+DESCRIPTION="Transform DocBook using TeX macros"
+HOMEPAGE="http://dblatex.sourceforge.net/"
+SRC_URI="mirror://sourceforge/${PN}/${P}py3.tar.bz2"
```

Comment:
You can just use the downloads... address, because this mirror doesn't do much better than that.